api/api_test.go

```diff
@@ -4,18 +4,16 @@ import (
 	"context"
 	"testing"
 
-	"github.com/ONSdigital/dp-search-reindex-api/store"
+	"github.com/ONSdigital/dp-search-reindex-api/mongo"
 	"github.com/gorilla/mux"
 	. "github.com/smartystreets/goconvey/convey"
 	"net/http/httptest"
 )
 
 func TestSetup(t *testing.T) {
 	Convey("Given an API instance", t, func() {
-		r := mux.NewRouter()
-		ctx := context.Background()
-		js := &store.DataStore{}
-		api := Setup(ctx, r, js)
+
+		api := Setup(context.Background(), mux.NewRouter(), &mongo.JobStore{})
```

Comment:
I'd use a mock of the jobstore (api/mock.JobStorerMock) rather than the actual mongo package (sticking to the ideal of api not knowing anything about mongo specifically)

Reply:
good idea - done